src/seedu/addressbook/common/Utils.java

```diff
@@ -35,4 +37,17 @@ public static boolean elementsAreUnique(Collection<?> items) {
         }
         return true;
     }
+    
+    /**
+     * Concatenates some printable object's printable representation together into a printable string
+     * @param printables
+     * @return concatenated printables' printable string  
+     */
+    public static String getPrintableStrings(Printable ...printables) {
+        StringBuilder sb = new StringBuilder();
+        for (Printable printable: printables) {
+            sb.append(printable.getPrintableString()).append(' ');
+        }
+        return sb.toString().trim();
```

Comment:
You could have used `StringJoiner` or `String.join` to concat, which avoids this trimming.